frontend/package.json

```diff
@@ -9,65 +9,59 @@
     "preview": "vite preview",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
-    "lint": "eslint . ",
-    "lint:fix": "prettier --write . && eslint --fix . ",
-    "format": "prettier --write .",
-    "test": "vitest run tests",
-    "test:watch": "vitest tests",
-    "coverage": "vitest run tests --coverage",
-    "prettier:check": "prettier --check .",
-    "algorithm-example": "ts-node-esm --experimentalSpecifierResolution node src/matching/examples/example.ts",
-    "generateTranslationKeyType": "node --no-warnings=ExperimentalWarning --loader ts-node/esm tools/translationKey/generateTranslationKeyType.ts && prettier --write src/lib/types/generated/translationKey.ts"
-  },
-  "lint-staged": {
-    "**/*.{svelte,css,html,js,json,md,ts,xml,yaml,yml}": [
-      "prettier --write",
-      "eslint"
-    ]
+    "algorithm-example": "tsx src/matching/examples/example.ts",
+    "generate:translation-key-type": "tsx tools/translationKey/generateTranslationKeyType.ts && prettier --write src/lib/types/generated/translationKey.ts"
```

Comment:
Rename yarn scripts to kebab-case and harmonizes (though not completely) usage of colons in script names to conform to the behaviour or coming versions of yarn where any script with a colon can be run globally from within any workspace.